src/tf/tf.cpp

```diff
@@ -1229,7 +1342,7 @@ struct tf_parser
             return literal{{shape_type}, data};
         return literal{{shape_type, dims}, data};
     }
-};
+}; // namespace MIGRAPHX_INLINE_NS
```

Comment:
There shouldn't be a semicolon after a namespace closes.

Reply:
fixed